### PR TITLE
AI/ML Homepage banner: updates for actual-hoc

### PR DESCRIPTION
**Pull request by molly-moen**
Update AI/ML homepage banner to have minor changes when hoc_mode is set to actual-hoc. The banner will update to say "The Hour of Code is here!" and the Join Us button will direct to the hour of code overview page:
Screen Shot 2020-11-30 at 1 55 03 PM

I also made a minor update to our button selection logic for banners based on Clare's comment on [this pr](https://github.com/code-dot-org/code-dot-org/pull/38005/files#r530018886).

## Links

- [jira](https://codedotorg.atlassian.net/browse/PLC-1062)

## Testing story
Tested locally in both actual-hoc and soon-hoc modes.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@
   social_hoc2020_global_movement: 'The Hour of Code is a global movement reaching tens of millions of students. One-hour tutorials are available in 45+ languages for all ages.'
   social_hoc2020_coming_dates: 'The Hour of Code is coming December 7-13, 2020. Computer science is foundational for every student to learn.'
   hoc2020_ai_subheader: 'The Hour of Code 2020 is coming!'
+  hoc2020_ai_subheader_actual_hoc: 'The Hour of Code 2020 is here!'
```
Comment (Hamms) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: note that we will not be able to get this string translated in time for HOC

Comment (molly-moen) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: 👍 we are aware

**Review by tess323 — APPROVED** [review]
LGTM!

**Review by bencodeorg — APPROVED** [review]

File: pegasus/src/homepage.rb [hunk 2]
```diff
@@ -105,9 +105,7 @@ def self.get_heroes
   end
 
   def self.get_actions(request)
-    code_break_takeover = promote_code_break(request)
-    code_bytes_takeover = promote_code_bytes(request)
-    hoc2020_ai_takeover = promote_hoc2020_ai(request)
+    hero_type = show_single_hero(request)
```
Comment (bencodeorg) [inline pegasus/src/homepage.rb:108]: This seems like a nice bit of cleanup :) Do we need to keep the `promote_` methods around?

Comment (bencodeorg) [reply pegasus/src/homepage.rb:108]: Oh...yes you do. Nvm!

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (molly-moen) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: we ended up using an existing string and doing away with the '2020' so it will be translated.

**Pull request merged**